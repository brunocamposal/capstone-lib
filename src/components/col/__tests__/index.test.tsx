import React from 'react'
import Col from '../index'
import renderer from 'react-test-renderer'

it('renderer', () => {
  const tree = renderer
    .create(
      <Col width={12} backgroundColor='white'>
        teste
      </Col>
    )
    .toJSON()

  expect(tree).toMatchSnapshot()
})
