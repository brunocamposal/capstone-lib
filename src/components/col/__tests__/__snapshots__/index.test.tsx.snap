// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renderer 1`] = `
<div
  className="sc-bdnylx jTSfH"
  width={12}
>
  teste
</div>
`;
